src/modality-specific-files/magnetoencephalography.md

```diff
@@ -546,7 +546,7 @@ A guide for using macros can be found at
 
 ### Example 2
 
-One recording per session, stored within the session directory.
+One empty-room recording per each participant's session, stored within the session directory.
```

Comment:
in BIDS, a session is always "per participant" (that is, you would not have two participants listed in the same session directory)

```suggestion
One empty-room recording per session, stored within the session directory.
```

Reply:
This was to clarify that in the other example (1) you can store under one session the noise recording to be used for multiple participants, but it's true it is redundant. How to store the emptyroom recordings is the thing I typically get the more questions about, as it is very MEG specific not applicable to other modalities, that's why I might have been over redundant in many places :)

```suggestion
One empty-room recording per session, stored within the participant's session directory.
```

Reply:
okay, let me try to push a commit and we can see whether you think that's an acceptable improvement.